cockatrice/src/window_main.cpp

```diff
@@ -107,12 +107,15 @@ void MainWindow::processConnectionClosedEvent(const Event_ConnectionClosed &even
         case Event_ConnectionClosed::LOGGEDINELSEWERE: reasonStr = tr("You have been logged out due to logging in at another location."); break;
         default: reasonStr = QString::fromStdString(event.reason_str());
     }
-    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.\nReason: %1").arg(reasonStr));
+    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.") + QString("<br>")
```

Comment:
I don't know which ones are problematic and which not...
But yes, the `reason` comes from the server so there is a vector. Good point.

So basically if there is one single html piece in all `QMessageBox` or 
`serverShutdownMessageBox.setInformativeText` etc. - even if the part containing it (here a `QString("<br>")`) has no arguments itself inside where the code could pop up - it's an potential problem?
So if one single subpart/substring of e.g. `QMessageBox` has html, the complete `QMessageBox` will be treated as such? @ctrlaltca 

Should we just exchange every single `<br>` in general?
And also add it to contributing guidelines with explanation?